src/app/charts/row-chart/row-chart-view/row-chart-view.component.ts

```diff
@@ -400,28 +403,29 @@ export class RowChartViewComponent implements OnInit, OnDestroy, AfterViewInit,
     // Moves all text in textGroups to a new HTML element
     moveTextToElement(chart: dc.RowChart, el: HTMLElement, vSpacing: number = 0) {
         const self = this;
-        const newSvg = d3.select(el).append('svg');
-        const textGroup = newSvg.append('g').attr('class', 'textGroup');
-
-        // Remove the old texts and append to the new group
-        const allText = chart.selectAll('g.row text');
-        const removed = allText.remove();
-        removed['nodes']().forEach((n) => {
-            textGroup.append(function() {
-                return n;
-            });
+        const container = d3.select(el).html('');
+        const svg = container.append('svg');
+        const group = svg.append('g').attr('class', 'textGroup');
+        const texts: any = chart.selectAll('g.row > text');
+
+        texts.each(function() {
+            this.style.display = 'none';
```

Comment:
Looks like it used to remove these nodes from the DOM, and now it hides them.  I think this is fine for this case, but just something to note.

Reply:
Thank you, I removed the unused code

Reply:
Yes I believe I had to do that otherwise the text values are missing on refresh